server/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "redis-shopping-cart-backend",
+    "version": "1.0.0",
+    "description": "",
```

Comment:
add here some description about using some redis functions - list them (I mean the new one)

Reply:
done